### DISCUSSION TITLE
page

### DISCUSSION THREAD

**Discussion by giscus** [Yiov/vitepress-doc#18] [General]
# page

我的vitpress文档教程

https://vitepress.yiov.top/page.html

**Comment by gaoziman** [comment]
如何使用阿里的图标库呢icon图标非svg

**Reply by Yiov** [reply]
非svg那就png啊，不是都可以用吗，具体用在哪个上面

**Comment by plmsmile** [comment]
你好，不同的分组，怎么设置不同的侧边栏呢？类似于这个博主这样：

https://xinlei3166.github.io/blog/Docker/Docker%E6%96%87%E6%A1%A3

https://hanochma.github.io/blogs/

**Reply by plmsmile** [reply]
oo，我知道了，多侧边栏配置。

**Comment by caolib** [comment]
好像这个网站有的图标都能用 https://simpleicons.org/

**Comment by xiaotimo** [comment]
你好，我想把很多内容放一个md文件中，他默认2-6级标题在右边，如何放在左边呢？相当于把右边的菜单当做左边的sidebar去使用

**Reply by Yiov** [reply]
其实 [Frontmatter教程](https://vitepress.yiov.top/frontmatter.html#%E4%BE%A7%E8%BE%B9%E6%A0%8F) 里都有的，单个md页面就关闭侧边栏，然后大纲栏靠左就行了

```
---
sidebar: false
aside: left
---
```

整个都改变就需要到 `config.mts` 中配置

```
themeConfig: {
    sidebar: false,
    aside:  'left',
}
```

**Comment by xiaotimo** [comment]
谢谢大佬




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2025年10月11日(星期六) 下午3:24
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Yiov/vitepress-doc] page (Discussion #18)














 
其实 Frontmatter教程 里都有的，单个md页面就关闭侧边栏，然后大纲栏靠左就行了
 --- sidebar: false aside: left ---  
整个都改变就需要到 config.mts 中配置
 themeConfig: {     sidebar: false,     aside:  'left', }
